db/schema/item.ts

```diff
@@ -1,23 +1,17 @@
 import mongoose, { Schema } from 'mongoose';
 
-import transaction from './transaction';
 import user from './user';
 
 const itemSchema: Schema = new mongoose.Schema({
-  uuid: String,
-  transaction: transaction,
+  id: String,
   currency: {
     type: String,
+    enum: ['USD', 'JPY', 'MXN', 'KRW', 'EUR'],
     default: 'USD',
   },
-  itemName: String,
+  name: String,
   people: [user],

```

Comment:
if we're using an array of users here, we should do the same for transaction. i don't really care either way since we're not saving anything but if we do want to validate the inputs with these schemas, might make more sense to keep this as an array of string for now.

Reply:
oh nah this one was my bad, i must've missed it in the schema - though it does lead to wonder how user accounts will be involved in the overall schema but that's something for future us to think about

Reply:
in the future, people will probably be an array of user, like how we have here. but yea for now, we don't need it since we don't have user registration, haha.